### DISCUSSION TITLE
How to make the opponent voice files and the bf voice files different

### DISCUSSION THREAD

**Discussion by libracrumbs** [KadeArchive/Kade-Engine#2264] [Q&A] [answered]
Because if you miss the opponents voice will be skipped too
So i need to code it different too

**Comment by glisteningtwilights** [comment]
Sorry I'm not sure how to do this, but I sure wish you the best of luck on finding someone who can help you. 😎😎😎

**Comment by Jotaro-Gaming** [comment, marked as answer]
> Because if you miss the opponents voice will be skipped too
> So i need to code it different too

make the opponent voice in the instrument and make 2 seperaet files named opponent without inst and inst without opponent so it can still  be used to make covers good luck on ur mod btw
